app/templates/index.html

```diff
@@ -2,91 +2,90 @@
 
 
 {% block pre_content %}
-{% for promo in content['promos'] %}
-<div class="alert alert-success d-flex align-items-center alert-dismissible fade show promo" role="alert">
-  <div style="margin-left: 10px">
-    <i class="fas fa-heart"></i>
-    {{promo.code}} <br/> {{promo.description}}
-  </div>
-  <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-</div>
-{% endfor %}
+  {% for promo in content['promos'] %}
+    <div class="alert alert-success d-flex align-items-center alert-dismissible fade show promo" role="alert">
+      <div style="margin-left: 10px">
+        <i class="fas fa-heart"></i>
+        {{promo.code}} <br/> {{promo.description}}
+      </div>
+      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+    </div>
+  {% endfor %}
 {% endblock %}
 
 
 {% block content %}
-
-<div class="row">
+  <div class="row">
     {% for service in content['services'] %}
-    <div class="col-md-4 col-sm-12 col-12">
-      <div class="card mb-4 shadow bg-white rounded">
-        <img src="{{ service.cover }}" class="card-img-top" alt="{{ service.title }}" height="225" width="100%">
-
-        <div class="card-body">
-          <h1><a href="/{{ service.slug }}.html">{{service.name}}</a></h1>
-          <p class="card-text">
-            {{ service.summary|striptags }}
-          </p>
-          <div class="d-flex justify-content-between align-items-center">
-            <div class="btn-group">
-              <a href="/{{ service.slug }}.html" class="btn btn-sm btn-outline-primary">
-                Details
-              </a>
+      <div class="col-md-4 col-sm-12 col-12">
+        <div class="card mb-4 shadow bg-white rounded">
+          <img src="/static/images/{{ service.images[0] }}" class="card-img-top" alt="{{ service.title }}" height="225" width="100%">
+
+          <div class="card-body">
+            <h1><a href="/{{ service.slug }}.html">{{service.name}}</a></h1>
+            <p class="card-text">
+              {{ service.summary|striptags }}
```

Comment:
hmmm.. do we have a `summary` field in the service? I think we can instead use the following: 
```suggestion
             {{ service.description | truncate(255, True) }}
```

This will truncate the service description to 255 chars.

Reply:
great idea, let me check it

Reply:
This has fixed